frontend/src/components/common/TooltipWrapper.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
-import { Overlay, Tooltip, OverlayTrigger } from 'react-bootstrap';
+import { Tooltip, OverlayTrigger, OverlayTriggerProps } from 'react-bootstrap';
 
-interface ITooltipWrapperProps extends Partial<React.ComponentPropsWithRef<typeof Overlay>> {
+interface ITooltipWrapperProps extends Partial<OverlayTriggerProps> {
```

Comment:
I was overriding the wrong base props here, this change ensures that all of the base overlaytriggerprops are passed to the underlying component.